### ISSUE TITLE
persianNumbersConverter

### ISSUE THREAD

**Issue by saeed74** [Hesammousavi/PersianAdminLTE#6]
Hi, i think persianNumbersConverter method (in demo.js file) is not a good way to show farsi numbers and make some problems. if someone use `<textarea></textarea>` and load some data from database and save it again on db it change the numbers data.

its better to use vazir farsi num hard coded font in places we need to show farsi number.

**Comment by ZahraReyhanian** [comment]
Yes , I have some problems with it. Specially when I use Ckeditor and upload images then I want to edit it, this method converts all numbers in address of images into Persian numbers and the images didn't load.

**Comment by lovelyroz** [comment]
in case of solve this problem, you must rename images then save it
